cl/users/views.py

```diff
@@ -201,27 +220,37 @@ def view_deleted_visualizations(
     return render(
         request,
         "profile/visualizations_deleted.html",
-        {"results": paged_vizes, "private": True},
+        {"results": paged_vizes, "page": "trash", "private": True},
     )
 
 
 @login_required
 @never_cache
 def view_api(request: AuthenticatedHttpRequest) -> HttpResponse:
-    if request.method == "POST":
-        instance = Webhook()
-        form = WebhookForm(request.POST, instance=instance)
-        if form.is_valid():
-            instance.user = request.user
-            form.save()
-            return HttpResponseRedirect(reverse("view_api"))
-    else:
-        form = WebhookForm()
+    return render(
+        request,
+        "profile/api.html",
+        {"private": True, "page": "info"},
+    )
 
+
+@login_required
+@never_cache
+def view_api_token(request: AuthenticatedHttpRequest) -> HttpResponse:
     return render(
         request,
         "profile/api.html",
-        {"webhook_form": form, "private": True},
+        {"private": True, "page": "token"},
```

Comment:
`api_token`? Sorry, I'll stop commenting on each one, but I think it's probably worth namespacing each just a bit, as I've been suggesting.

Reply:
Yeah, I also changed these on profile views to have the same format, e.g: `profile_donations`